src/rlx_util.erl

```diff
@@ -173,7 +173,8 @@ render(Template, Data) when is_list(Template) ->
     render(ec_cnv:to_binary(Template), Data);
 render(Template, Data) when is_binary(Template) ->
     case catch bbmustache:render(Template, Data,
-                                 [{key_type, atom}]) of
+                                 [{key_type, atom},
+                                  {escape_fun, fun(X) -> X end}]) of
```

Comment:
Why change the escape function? Isn't this altering existing functionality, or was bbmustache not escaping before?

There should be the triple bracket template for non-escaped values, `{{{key}}}`.

Reply:
yeah, i guess it's kinda hard on existing functionality, what about an option to enable this globally? It gets kinda old always having to remember use of triple brackets, most people aren't expecting any kind of escaping to happen on their sys.config files.

Reply:
What about having all the templates we provide use triple brackets? Sort of moving towards that being the one everyone uses by default and instead they have to remember that to escape they need double brackets :).

Reply:
which templates? you mean the default `sys_config` and `vm_args`?

Reply:
Yea, all these use templates: vm_args, bin, bin_windows, erl_ini, erl_script, extended_bin, extended_bin_windows

Reply:
my guess is that no one would really notice that, this triple bracket business also applies when you use the overlay `template` command, i think that's where most people are bit by this as i think was the case in erlware/relx##497

Reply:
Sure, I just mean it is simply the issue of using `{{{` instead of `{{` so if all our docs and templates and examples do that it will just become what people use. It just seems off to change mustache defined behaviour when it fully supports this.

Reply:
fair enough, i'm closing this then as there is no real added value in bumping moustache